pkg/logs/input/container/launcher.go

```diff
@@ -24,26 +24,18 @@ import (
 // the launcher will attempt to initialize a kubernetes launcher which will detect and tail all the logs files localized
 // in '/var/log/pods' of all the containers running on the kubernetes cluster.
 func NewLauncher(collectAll bool, sources *config.LogSources, services *service.Services, pipelineProvider pipeline.Provider, registry auditor.Registry) restart.Restartable {
-	switch {
```

Comment:
Do we want to add a log that explains the strategy here ? i.e. try docker, fallback to kubernetes, finally noop ?